MNE/disp3D/model/items/sourceactivity/ecddatatreeitem.cpp

```diff
@@ -204,23 +204,26 @@ void EcdDataTreeItem::setVisible(bool state)
 void EcdDataTreeItem::plotDipoles(QSharedPointer<ECDSet> pECDSet)
 {
     //Plot dipoles
-    QVector3D pos,to,from;
+    QVector3D pos, to, from;
+
+    //The Qt3D default cone orientation and the top of the cone lies in line with the positive y-axis.
+    from = QVector3D(0.0, 1.0, 0.0);
+    double norm;
 
     for(int i = 0; i < pECDSet->size(); ++i) {
         pos.setX((*pECDSet)[i].rd(0));
         pos.setY((*pECDSet)[i].rd(1));
         pos.setZ((*pECDSet)[i].rd(2));
 
-        to.setX((*pECDSet)[i].Q(0));
-        to.setY((*pECDSet)[i].Q(1));
-        to.setZ((*pECDSet)[i].Q(2));
-        //to.normalize();
+        norm = sqrt(pow((*pECDSet)[i].Q(0),2)+pow((*pECDSet)[i].Q(1),2)+pow((*pECDSet)[i].Q(2),2));
```

Comment:
could you use the norm to scale the arrow up and down - like 1 nAm equals 1cm or something like this